ibm/service/contextbasedrestrictions/resource_ibm_cbr_zone.go

```diff
@@ -212,23 +233,31 @@ func ResourceIBMCbrZoneValidator() *validate.ResourceValidator {
 			MinValueLength:             0,
 			MaxValueLength:             300,
 		},
+		validate.ValidateSchema{
+			Identifier:                 "x_correlation_id",
+			ValidateFunctionIdentifier: validate.ValidateRegexpLen,
+			Type:                       validate.TypeString,
+			Optional:                   true,
+			Regexp:                     `^[a-zA-Z0-9 ,\-_]+$`,
+			MinValueLength:             1,
+			MaxValueLength:             1024,
+		},
+		validate.ValidateSchema{
+			Identifier:                 "transaction_id",
+			ValidateFunctionIdentifier: validate.ValidateRegexpLen,
+			Type:                       validate.TypeString,
+			Optional:                   true,
+			Regexp:                     `^[a-zA-Z0-9 ,\-_]+$`,
+			MinValueLength:             1,
+			MaxValueLength:             1024,
+		},
 	)
 
 	resourceValidator := validate.ResourceValidator{ResourceName: "ibm_cbr_zone", Schema: validateSchema}
 	return &resourceValidator
 }
 
-func getIBMCbrAccountId(meta interface{}) (string, error) {
```

Comment:
why we removed this ? We added accoutn_id as optional...will this work if user doens't send account_id

Reply:
This is removed by code generator. 
This function is not called by nowhere